test/system/500-networking.bats

```diff
@@ -767,4 +767,14 @@ EOF
     is "$output" "" "Should print no output"
 }
 
+@test "podman network rm --dns-option " {
+    dns_opt=dns$(random_string)
+    run_podman run --rm --dns-opt=${dns_opt} $IMAGE cat /etc/resolv.conf
+    is "$output" ".*options ${dns_opt}" "--dns-opt was added"
+
+    dns_opt=dns$(random_string)
+    run_podman run --rm --dns-option=${dns_opt} $IMAGE cat /etc/resolv.conf
```

Comment:
It took me several seconds of concentration to figure out that the difference is `opt` vs `option`. This may cost someone else time, too. If you need to repush for any reason, would you consider:
```bash
    for opt in dns-opt dns-option; do
        dns_value=...
        run_podman run --rm --$opt=${dns_value} ...
        is ... "--$opt was added"
```